test/watcher.test.js

```diff
@@ -97,7 +97,7 @@ jest.setTimeout(30000);
 
 it('Should support custom watch API', async () => {
   let buildCnt = 0;
-  const buildFile = path.resolve('./test/integration/request.js');
+  const buildFile = path.resolve('./test/integration/twilio.js');
```

Comment:
Does it work work with `request` now that `node-libs-browser` is added?

Reply:
request.js will work for both whatever you add the nodejs lib fallback support or not. twilio.js doesn't work without fallback that breaks the watcher test case.

I changed the original file from twilio to request to let it pass the test in last commit.